lithium/lithium.py

```diff
@@ -207,10 +214,12 @@ def addArgs(self, parser):
         grp_add.add_argument(
             "--chunkstart", type=int,
             default=0,
-            help="For the first round only, start n chars/lines into the file. Best for max to divide n. [Mostly intended for internal use]")
+            help="For the first round only, start n chars/lines into the file. Best for max to divide n. "
+            "[Mostly intended for internal use]")
```

Comment:
I'll change this now that the linter doesn't complain about the latter. (not sure why it did previously)